src/main/java/com/github/osisoft/dataviewsample/App.java

```diff
@@ -300,13 +300,48 @@ public static Boolean toRun() {
             System.out.println(dataViewStoredData);
             assert dataViewStoredData.length() > 0 : "Error getting data view stored data";
 
+            // Step 14
+            System.out.println();
+            System.out.println("Step 13: Demonstrate accept-verbosity header usage");
+
+            System.out.println("Writing default values to one property of the stream");
+            // Keep the times in the future, guaranteeing no overlaps with existing data
+            Instant default_data_start_time = Instant.now().plus(Duration.ofHours(1));
+            Instant default_data_end_time = default_data_start_time.plus(Duration.ofHours(1));
+
+            // The values are only a pressure, keeping temperature as 0 (the default value of a non-nullable double data type)
+            String values = "[{\"time\": \"" + default_data_start_time + "\", \"pressure\": 100, \"temperature\":0},{\"time\": \"" + default_data_end_time + "\", \"pressure\": 50, \"temperature\":0}]";
```

Comment:
is there a better want to do this string...?

Reply:
I'm mainly basing it off of the way that the sample writes values to the streams in the beginning [lines 394-406](https://github.com/osisoft/sample-adh-data_views-java/blob/3f74dd1089b32e174d24e78127d5a74749749b9c/src/main/java/com/github/osisoft/dataviewsample/App.java#L394). The json string is manually built, but one value at a time; then all the strings are joined together with commas and wrapped in square brackets. 

So this approach here just throws all of that into one line. There's almost definitely a better way to do it, but this at least is consistent within the sample, for what that's worth.

Reply:
i just changed it so that it does match exactly how the initial writes are done. it's cleaner and easier to read, but it's still manual json string creation. If we want to do something better than that (like having the gson library serialize it for us?), I'd have to look into that to find the best way, and I'd want to change the other section too for consistency.